services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = get_user_model().objects.get(username=username)

```

Comment:
It's recommended to use `get_user_model()` instead of `User` in your code. This is a best practice that ensures your code will always use the correct user model, even if it's been overridden in your project's settings.

Reply:
I've used get_user_model() here